conf/messages

```diff
@@ -146,15 +146,15 @@ taxYears.oldSa.agent.content.3								   = This will be a different Government G
 tax-year-summary.heading                                           = Tax year summary
 tax-year-summary.heading-secondary                                 = 6 April {0} to 5 April {1}
 tax-year-summary.calculation-date                                  = Calculation date
-tax-year-summary.calc-from-last-time                               = This is only based on figures we’ve already received to {0}. It’s not your final tax bill.
+tax-year-summary.calc-from-last-time                               = This is not your final tax bill, it''s only based on figures we''ve already received.
```

Comment:
two single quotes ('')? is this right?

Reply:
Thank's, I did exactly as design team want it. Double tick will render to just one. But you are right I've changed it to what we are using. Done